### DISCUSSION TITLE
Application error: a client-side exception has occurred (see the browser console for more information).

### DISCUSSION THREAD

**Discussion by rajanshresth** [pmndrs/zustand#1861] [General]
![image](https://github.com/pmndrs/zustand/assets/76089000/62d2e1b5-7250-4c2a-906d-a4a5bccda0be)

**Comment by dai-shi** [comment]
Is it https://docs.pmnd.rs/zustand/getting-started/comparison ?

It seems to work for me.
<img width="760" alt="image" src="https://github.com/pmndrs/zustand/assets/490574/38542546-6749-4ff9-acbe-0b2c04b9c219">
